doc/classes/@GDScript.xml

```diff
@@ -264,10 +264,14 @@
 			<argument index="0" name="step" type="float">
 			</argument>
 			<description>
-				Returns the position of the first non-zero digit, after the decimal point.
+				Deprecated alias for "step_decimals". Returns the position of the first non-zero digit, after the decimal point.
```

Comment:
You can use `[method step_decimals]` for a direct link to that method. I'd also remove the rest of the documentation for this one, only mention that it's deprecated by `step_decimals`.